openwisp_radius/templates/radius_accounting_start.html

```diff
@@ -2,4 +2,5 @@
 <div class="msg">
 <p>{% trans "A new session has been started for your account:" %} {{ user.get_username }}.</p>
 <p>{% trans "You can review your session to find out how much time and/or traffic has been used or you can terminate the session at any time by clicking on the button below." %}</p>
+{% if sesame_max_age %}<p>{% trans "NOTE: This link is valid only for" %} {{ sesame_max_age }} hrs</p> {% endif %}
```

Comment:
please use this form:
https://docs.djangoproject.com/en/4.0/topics/i18n/translation/#blocktranslate-template-tag

Different languages may place the `sesame_max_age` value before or after specific words, languages of our planet are very different from one another, we can't assume the value will be always displayed at the end of the phrase.

Reply:
> please use this form: https://docs.djangoproject.com/en/4.0/topics/i18n/translation/#blocktranslate-template-tag
> 
> Different languages may place the `sesame_max_age` value before or after specific words, languages of our planet are very different from one another, we can't assume the value will be always displayed at the end of the phrase.

You are right, I'll read docs and fix this 👍